packages/cozy-client/src/helpers.js

```diff
@@ -76,6 +78,47 @@ export const generateWebLink = ({
   return url.toString()
 }
 
+/**
+ * Deconstruct the given link in order to retrieve useful data like Cozy's name, domain, or slug
+ *
+ * The given link MUST contain a slug
+ *
+ * @param {string} webLink - link to deconstruct. It should be a link from a Cozy and containing a slug
+ * @param {SubdomainType} subDomainType - whether the cozy is using flat or nested subdomains. Defaults to flat.
+ * @returns {CozyLinkData} Deconstructed link
+ */
+export const deconstructCozyWebLinkWithSlug = (
```

Comment:
nit: Why no calling it simply `deconstructCozyWebLink`

Reply:
I wanted to emphasise the fact that this method only works if the link contains a slug.